xblock/plugin.py

```diff
@@ -120,12 +120,15 @@ def select(identifier, all_entry_points):
         return PLUGIN_CACHE[key]
 
     @classmethod
-    def load_classes(cls):
+    def load_classes(cls, fail_silently = True):
         """Load all the classes for a plugin.
 
         Produces a sequence containing the identifiers and their corresponding
         classes for all of the available instances of this plugin.
 
+        fail_silently causes the code to simply log warnings if a
+        plugin cannot import. I'm not sure why that's a good idea.
```

Comment:
"I'm not sure why that's a good idea" is bad text for a docstring.

Imagine a situation in which an XBlock depends on Django being configured (such as openassessment). You may want to use parts of such an XBlock, or libraries from that XBlock (and thus have it installed), but wouldn't want to completely break your app.